galaxy_importer/loaders/collection.py

```diff
@@ -88,6 +92,8 @@ def load(self):
         self.contents = self._build_contents_blob()
         self.docs_blob = self._build_docs_blob()
         self.requires_ansible = loaders.RuntimeFileLoader(self.path).get_requires_ansible()
+        if self.cfg.run_ansible_lint:
+            self._lint_collection()
```

Comment:
since this method logs text that is more open-ended and can have lots of output (and it is configurable), it might be best to call this at the end, after the more simple checks below.

Reply:
Agreed. :+1: I'll move it.

Reply:
This looks ok as-is